src/confcom/azext_confcom/container.py

```diff
@@ -367,8 +404,8 @@ def _get_environment_rules(self) -> List[Dict[str, Any]]:
             if rule[config.ACI_FIELD_CONTAINERS_ENVS_NAME] not in env_var_names:
                 out_rules.append(
                     {
-                        config.POLICY_FIELD_CONTAINERS_ELEMENTS_ENVS_RULE:
-                        f"{rule[config.ACI_FIELD_CONTAINERS_ENVS_NAME]}="
+                        # pylint: disable=line-too-long
+                        config.POLICY_FIELD_CONTAINERS_ELEMENTS_ENVS_RULE: f"{rule[config.ACI_FIELD_CONTAINERS_ENVS_NAME]}="
```

Comment:
does having a line break here not work? I think that would be nicer than disabling pylint

Reply:
I have this auto format and every time when I do line break it changes back to this.